plugins/lookup/filetree.py

```diff
@@ -41,6 +43,7 @@
     src: '{{ item.src }}'
     dest: /web/{{ item.path }}
     state: link
+    follow: false # avoid corrupting target files if the link already exists
```

Comment:
Just a tiny change: two spaces before `#` make it slightly better readable IMO.
```suggestion
    follow: false  # avoid corrupting target files if the link already exists
```